sable_network/src/network/event/details.rs

```diff
@@ -190,10 +190,14 @@ EventDetails => {
     }
 
     #[target_type(ServerId)]
-    struct IntroduceServices {
+    struct IntroduceServicesServer {
         pub sasl_mechanisms: Vec<String>,
     }
 
+    #[target_type(ServerId)]
+    struct IntroduceHistoryServer {
```

Comment:
I realise this is matching the services behaviour and it's probably fine for now, but long term the desired behaviour isn't the same - we need a single authoritative services node, but for history we can have multiple active nodes and just send requests to any of them.

Reply:
ack